src/components/HomePage/HomePage.tsx

```diff
@@ -0,0 +1,11 @@
+import { Outlet } from 'react-router-dom';
+
+export const HomePage = () => {
+  return (
+    <>
+      <h1 className="title">Home Page</h1>
+
+      <Outlet />
```

Comment:
I think you don't need this
An `<Outlet/>` should be used in `parent route elements` to render their child route elements